packages/tpetra/CMakeLists.txt

```diff
@@ -1106,6 +1106,16 @@ GLOBAL_SET(${PACKAGE_NAME}_ETI_DEVICES ${${PACKAGE_NAME}_ETI_DEVICES})
 GLOBAL_SET(${PACKAGE_NAME}_ETI_NODES   ${${PACKAGE_NAME}_ETI_NODES})
 
 
+
+# Experimental Code
+# Enable experimental code
+TRIBITS_ADD_OPTION_AND_DEFINE(
+  ${PACKAGE_NAME}_ENABLE_Experimental
+  HAVE_${PACKAGE_NAME_UC}_EXPERIMENTAL
+  "Enable experimental code."
+  NO)
```

Comment:
@csiefer2 this looks like a typo, should this be ON or OFF?

Reply:
That code is cut and pasted from MueLu, so I'm pretty sure Tribbles doesn't care...